foxglove_bridge_base/src/serialization.cpp

```diff
@@ -37,6 +38,11 @@ void to_json(nlohmann::json& j, const Parameter& p) {
     j["value"] = p.getValue<double>();
   } else if (paramType == ParameterType::PARAMETER_STRING) {
     j["value"] = p.getValue<std::string>();
+  } else if (paramType == ParameterType::PARAMETER_BYTE_ARRAY) {
+    const auto paramValue = p.getValue<std::vector<unsigned char>>();
+    const std::string strValue(paramValue.begin(), paramValue.end());
+    j["value"] = base64Encode(strValue);
```

Comment:
too bad we have to copy here, might be something to fix in a future PR?

Reply:
Fixed it by using `std::string_view`